spec/spec_helper.rb

```diff
@@ -14,3 +18,30 @@
 end
 
 WebMock.disable_net_connect!(allow_localhost: true, allow: "percy.io")
+
+class MockDashboard < Administrate::BaseDashboard
+  ATTRIBUTE_TYPES = {
+    name: Administrate::Field::String,
+    email: Administrate::Field::Email,
+    phone: Administrate::Field::Number,
+  }
+end
+
+class DashboardWithAnArrayOfScopes < Administrate::BaseDashboard
+  ATTRIBUTE_TYPES = {
+    name: Administrate::Field::String,
+  }
+
+  COLLECTION_SCOPES = [:active, :old, "with_argument(3)", "idle"]
+end
+
+class DashboardWithAHashOfScopes < Administrate::BaseDashboard
+  ATTRIBUTE_TYPES = {
+    name: Administrate::Field::String,
+  }
+
+  COLLECTION_SCOPES = {
+    status: [:active, :inactive, "idle"],
+    other: [:last_week, :old, "with_argument(3)",],
```

Comment:
Avoid comma after the last item of an array, unless each item is on its own line.